<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newbill</class>
 <widget class="QDialog" name="newbill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Statements</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="7">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Selected Statements</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Available Statements</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addStatement">
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeStatement">
       <property name="text">
        <string>REMOVE</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="7">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="autoButton">
       <property name="text">
        <string>Auto Compare</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="manualButton">
       <property name="text">
        <string>Manual Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="compareProgress">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="availableStatements"/>
   </item>
   <item row="1" column="7">
    <widget class="QListWidget" name="selectedStatements"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
